src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -278,9 +273,9 @@ export class zonePacketHandlers {
       client: Client,
       packet: any
     ) {
-      const characterId = packet.data.characterId;
-      const damage = packet.data.damage;
-      const vehicle = server._vehicles[characterId];
+      const characterId = packet.data.characterId,
+      damage = packet.data.damage,
```

Comment:
i've to admit that i'm not a huge fan of this syntax, i will just do a quick bench to be sure if it doesn't change anything ( it shouldn't )

Reply:
yep it doesn't change anything the only bad thing is  that if you mess up a comma like that the variable become a global one ( which is never cleaned from memory) but as long at it never happen it's good 👍